src/app/frontend/replicasetlist/replicasetlist.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+$replicaset-deploy-btn: #eb3a2e;
```

Comment:
Default accent is actually pink color. I'm not convinced about this look.

![zrzut ekranu z 2015-12-22 12-58-56](https://cloud.githubusercontent.com/assets/2285385/11954815/ea382ac2-a8ab-11e5-8896-8dbd73cc225c.png)

![zrzut ekranu z 2015-12-22 12-58-49](https://cloud.githubusercontent.com/assets/2285385/11954817/ebccfc14-a8ab-11e5-94b8-d1e9855c8d9c.png)

Maybe redefine accent to red palette? It looks a bit better I think.

![zrzut ekranu z 2015-12-22 13-04-03](https://cloud.githubusercontent.com/assets/2285385/11954873/8276755a-a8ac-11e5-8a83-af6ba0ae3d43.png)

![zrzut ekranu z 2015-12-22 13-04-09](https://cloud.githubusercontent.com/assets/2285385/11954874/83b48c86-a8ac-11e5-87d7-2fdd6f8e7c80.png)